src/ontology/cl-edit.owl

```diff
@@ -28289,23 +28285,23 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11884355") obo:IAO_00001
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023088 <http://orcid.org/0000-0001-7258-9596>)
 AnnotationAssertion(oboInOwl:inSubset obo:CL_4023088 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
 AnnotationAssertion(rdfs:label obo:CL_4023088 "large basket cell")
-SubClassOf(obo:CL_4023088 obo:CL_0000118)
+EquivalentClasses(obo:CL_4023088 ObjectIntersectionOf(obo:CL_0000118 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070004)))
 
 # Class: obo:CL_4023089 (nest basket cell)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11884355") obo:IAO_0000115 obo:CL_4023089 "A basket cell which has simpler dendritic arbors (compared to small or large basket cells), and an axonal plexus of intermediate density, composed of a few long, smooth axonal branches.")
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023089 <http://orcid.org/0000-0001-7258-9596>)
 AnnotationAssertion(oboInOwl:inSubset obo:CL_4023089 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
 AnnotationAssertion(rdfs:label obo:CL_4023089 "nest basket cell")
-SubClassOf(obo:CL_4023089 obo:CL_0000118)
+EquivalentClasses(obo:CL_4023089 ObjectIntersectionOf(obo:CL_0000118 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070005)))
```

Comment:
You should be able to use a more general CL class here and rely on inference from PATO hierarchy to classify under basket cell.